packages/vue/src/components/authenticator.vue

```diff
@@ -15,12 +15,11 @@ import {
   AuthenticatorRoute,
   SocialProvider,
   authenticatorTextUtil,
-  configureComponent,
+  setUserAgent,
 } from '@aws-amplify/ui';
 
 import { useAuth, useAuthenticator } from '../composables/useAuth';
 import { UseAuthenticator } from '../types';
-import { VERSION } from '../version';
```

Comment:
Not an official official style rule or anythiing, but we try to group imports in the following order:

_Framework/Framework 3pp_

```ts
import React from 'react';
import { something } from 'react-hook-form';
```

_Internal imports_
```ts
import { somethingBetter } from '@aws-amplify/ui-react-core';
```
_relative paths_
1) by relative proximity of the path
2) alphabetize by import path

```ts
import { stuff } from '../../stuff';
import { things } from '../../thngs';

import { nearby } from './nearby';
```